instructors/4-gis-slides.qmd

```diff
@@ -492,14 +528,22 @@ ggplot() +
 
 ## Intersection
 
-Now, we would like to distinguish conservation areas based on the number of historic buildings they contain. In GIS terms, we would like to know how many centroids each fused buffer polygon contains. 
+Now, we would like to distinguish conservation areas based on the number of historic buildings they contain.
+In GIS terms, we would like to know how many centroids each dissolved buffer polygon contains. 
 
-This operation means intersecting the layer of polygons with the layer of points and the corresponding function is `st_intersection()`.
+This operation means intersecting the polygon layer with the point layer.
+The corresponding function is `st_intersection()`.
+The corresponding function is `st_intersection()`.
```

Comment:
Repetition?

Reply:
Issue to be soleved in https://github.com/carpentries-incubator/r-geospatial-urban/issues/66